subprojects/instant-execution/src/integTest/groovy/org/gradle/instantexecution/InstantExecutionGroovyIntegrationTest.groovy

```diff
@@ -16,27 +16,30 @@
 
 package org.gradle.instantexecution
 
-
 import spock.lang.Ignore
 
+
 class InstantExecutionGroovyIntegrationTest extends AbstractInstantExecutionIntegrationTest {
 
     @Ignore
     def "instant execution for compileGroovy on Groovy project with no dependencies"() {
+
+        def instantExecution = new InstantExecutionBuildOperationsFixture(executer, temporaryFolder)
```

Comment:
❌ `AbstractInstantExecutionIntegrationTest` could still provide a factory method as before to avoid the duplication and spread of dependencies.